AdyenComponents/UPI/UPIComponentDetails.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+
+/// Contains the details supplied by the UPI based component.
+public struct UPIComponentDetails: PaymentMethodDetails {
+
+    @_spi(AdyenInternal)
+    public var checkoutAttemptId: String?
+
+    ///  Selected flow type
+    public let type: String
+
+    ///  entered vpa
```

Comment:
Minor: Should start with capital

Reply:
Fixed. Here is the commit: 32a3cdc9cc0d9e1ff71bdd9886176033095764dd